libr/core/cbin.c

```diff
@@ -199,7 +199,7 @@ static bool false_positive(const char *str) {
 	return false;
 }
 
-static bool strpurge(RCore *core, const char *str, ut64 refaddr) {
+R_API bool r_core_bin_strpurge(RCore *core, const char *str, ut64 refaddr) {
```

Comment:
seems like something to be moved into RBin instead, but ill merge for now

Reply:
yep, the entire string filter subsystem apparently should be in `libr/bin/filter.c`